Jenkinsfile

```diff
@@ -46,6 +46,8 @@ pipeline {
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_config_mttcs && /home/saluser/.checkout_repo.sh \${work_branches}\"
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_xml && /home/saluser/.checkout_repo.sh \${work_branches}\"
                     docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_idl && /home/saluser/.checkout_repo.sh \${work_branches}\"
+                    docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_sal && git reset --hard ab4b50c\"
+                    docker exec -u saluser \${container_name} sh -c \"source ~/.setup.sh && cd /home/saluser/repos/ts_xml && git reset --hard b93c844\"
```

Comment:
As above, please do:

```
source ~/.setup.sh && cd /home/saluser/repos/ts_xml && /home/saluser/.checkout_repo.sh \${work_branches}
```

Note that the above will catch those cases where you are developing a new xml, as long as you name the xml branch the same as the working branch.

Reply:
The code you showed is in the Jenkinsfile already. I added the specific commit is because the `ts_xml` `develop` branch is breaking the `ts_salobj` for the `ts_xml` commit: `f6087af`. I documented this in the `version_history.rst` already.

Reply:
Done in commit: b085919. Thanks!